packages/phone-number-privacy/signer/migrations/20200330212224_create-accounts-table.ts

```diff
@@ -1,4 +1,4 @@
-import * as Knex from 'knex'
+import Knex from 'knex'
```

Comment:
why change these? In theory they're equivalent but just a bit wary to change since knex itself auto-gened most of the content in these files

Reply:
Yea I'm actually facing an issue with this rn. I'll ping you to discuss